src/Bookings.js

```diff
@@ -1,21 +1,51 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
+import SearchResult from "./component/SearchResult.js";
 // import FakeBookings from "./data/fakeBookings.json";
-
+import CustomerProfile from "./CustomerProfile.js";
 const Bookings = () => {
+  const [bookings, setBookigs] = useState([]);
+  const [customerProfile, setCustomerProfile] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
   const search = searchVal => {
-    console.info("TO DO!", searchVal);
+    const searchValue = searchVal.toLowerCase();
+    setBookigs(
+      bookings.filter(element => {
+        if (
+          searchValue === "" ||
+          element.firstName.toLowerCase().includes(searchValue) ||
```

Comment:
Don't forget that the search value should also be `.toLowerCase()`. Otherwise it will only match when the user types in the search box only with lower case